README.md

```diff
@@ -145,7 +185,7 @@ code, include a script or makefile that will do the job.
 
 ## A2.5
 
-«replace this with your answer»
+node A2-5.js
```

Comment:
First, thank you for writing tests, and including both positive and negative tests.

I added a couple of my own, and confirmed that your parser doesn't support repeating adjectives "the smelly smelly dog"

 I also appreciated that you used a real table to drive it.

However, because your original state machine wasn't really a state machine, you ran into problems, and ended up putting parsing logic in the code, too:

~~~ js
                if (nextState.length == 2
                    &&
                    (words[index+1] == "dog"
                    || words[index+1] == "cat"))
                    currentState = nextState[1]
                else
                    currentState = nextState[0]
                break
~~~

# -4